janusgraph-server/src/test/resources/empty-sample.groovy

```diff
@@ -0,0 +1,32 @@
+import org.apache.tinkerpop.gremlin.server.util.LifeCycleHook
```

Comment:
Is this file needed?

Reply:
Yes, otherwise I got `The traversal source [g] for alias [g] is not configured on the server` error. Not sure if there's a better way to solve this.

Reply:
Will merge this for now. If anyone has a better idea on handling the error, let me know or create a PR.